src/compiler/types.ts

```diff
@@ -1693,6 +1693,8 @@ namespace ts {
           * @param path The path to test.
           */
         fileExists(path: string): boolean;
+
+        readFile(path: string): string;

```

Comment:
@mhegazy If it is part of 2.1 on `typescript@next` we will have plenty of time to react and this should not be significant for us.

@weswigham This can be better but we do this, in general, to make testing easier.